src/components/SkillComboBox/SkillComboBox.tsx

```diff
@@ -267,16 +268,7 @@ const AddItem = ({
         css={{ justifyContent: 'space-between', width: '100%', gap: '$lg' }}
       >
         <Text>Be the First to Add</Text>
```

Comment:
Does moving this to bottom change ordering? Maybe Graffe can help investigate why ordering isnt working as expected?

Reply:
it doesn't change ordering as far as I can tell.  i'm not able to reproduce the bug locally